static/app/components/superuserAccessForm.tsx

```diff
@@ -0,0 +1,73 @@
+import {Component} from 'react';
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import Form from 'sentry/components/forms/form';
+import Hook from 'sentry/components/hook';
+import ThemeAndStyleProvider from 'sentry/components/themeAndStyleProvider';
+import {ErrorCodes} from 'sentry/constants/superuserAccessErrors';
+import {IconFlag} from 'sentry/icons';
+import {t} from 'sentry/locale';
+
+type State = {
+  error: boolean;
+  errorType: string;
+};
+
+class SuperuserAccessForm extends Component<State> {
+  state: State = {
+    error: false,
+    errorType: '',
+  };
+
+  handleSuccess = () => {
+    window.location.reload();
```

Comment:
huh
I guess if they've successfully auth'd they'll have their session token and refresh will redirect them?
Would we prefer a `window.location.replace` or was there a reason we wanted the server to handle the redirect? I can't remember what we chatted about lol

Reply:
I figured since they're already on the route that we wanted to redirect them to, so reloading the page would take them to where we needed them to go. With `window.location.replace` i need the url of where to redirect but that would require me make public the route from getsentry